linera-views/src/lru_caching.rs

```diff
@@ -24,12 +24,12 @@ use {
     async_lock::MutexGuardArc,
 };
 
-/// The LruPrefixCache store the data for a simple read_keys queries
-/// It is inspired from lru-cache crate.
+/// The LruPrefixCache stores the data for simple read_keys queries
+/// It is inspired by the crate `lru-cache`.
 ///
 /// We cannot apply this crate directly because the batch operation
-/// need to update the cache. In the case of DeletePrefix we have to
-/// handle the keys by prefix. And so we need to have a BTreeMap to
+/// need to update the cache. In the case of `DeletePrefix` we have to
+/// handle the keys by prefixes. And so we need to have a BTreeMap to
```

Comment:
I actually think "keys by prefix" sounds fine. (But maybe it's not correct. :shrug:)

Reply:
I prefer the prefixes since we can do several deletions by prefixes.